.gitignore

```diff
@@ -58,7 +58,7 @@
 /ntp/ntp-4.2.8p15-32
 /openldap/openldap-2.5.14-32
 /openlldp/openlldp-0.4alpha-32
-/openssh/openssh-9.8p1-32
```

Comment:
NOTE:  A `git status` showed me I forgot to update this with the last OpenSSH update.